From d8681d9e8755f5c14eaedf1959d8e6d073406be2 Mon Sep 17 00:00:00 2001
From: "qiang.cheng" <qiang.cheng@amlogic.com>
Date: Thu, 3 Aug 2023 17:26:49 +0800
Subject: [PATCH] debian: install libreadline and libtinfo for bluetooth [1/1]

PD#SWPL-132450

Problem:
BT transferFile(send) fail when exec obexctl

Solution:
install libreadline and libtinfo so

Verify:
local

Signed-off-by: qiang.cheng <qiang.cheng@amlogic.com>
Change-Id: I7c3d0b7a41b3259d7482a1248c32b5271bf0af41
---
 env/copy_debs_from_yocto.sh | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/env/copy_debs_from_yocto.sh b/env/copy_debs_from_yocto.sh
index 217d661..484f047 100755
--- a/env/copy_debs_from_yocto.sh
+++ b/env/copy_debs_from_yocto.sh
@@ -65,7 +65,6 @@ package_status_append=(kernel-5.15.78_5.15.78+git999-r2_arm64.deb \
 			liborc-0.4-0_0.4.31-r0_arm64.deb \
 			libpanelw5_6.2-r0_arm64.deb \
 			libpulse0_13.0-r0_arm64.deb \
-			libreadline5_5.2-r9_arm64.deb \
 			librsvg-2-2_2.40.21-r0_arm64.deb \
 			libsamplerate0_0.1.9-r1_arm64.deb \
 			libsbc1_1.4-r0_arm64.deb \
@@ -73,7 +72,6 @@ package_status_append=(kernel-5.15.78_5.15.78+git999-r2_arm64.deb \
 			libsoup-2.4_2.68.4-r0_arm64.deb \
 			libspeex1_1.2.0-r0_arm64.deb \
 			libtag1_1.11.1-r0_arm64.deb \
-			libtinfo5_6.2-r0_arm64.deb \
 			libwebp_1.1.0-r0_arm64.deb \
 			libxml2_2.9.10-r0_arm64.deb \
 			libz1_1.2.11-r0_arm64.deb \
@@ -104,7 +102,7 @@ package_status_append=(kernel-5.15.78_5.15.78+git999-r2_arm64.deb \
 			alsa-plugins-pulseaudio-conf_1.2.1-r0_arm64.deb)
 
 packages_arr=(  gst-aml-imgproc gst-aml-ion-allocator gst-aml-gfx2d gst-plugin-conv \
-                gst-plugin-nn gst-plugin-overlay detect-library \
+                readline ncurses gst-plugin-nn gst-plugin-overlay detect-library \
                 npu-common alsa-lib alsa-state alsa-utils alsa-plugins alsa-utils-scripts aml-amaudioutils aml-audio-hal \
                 aml-audio-service aml-avsync aml-cas-hal aml-dtvdemod aml-dvb \
                 aml-dvbaudioutils aml-hdcp aml-hdcp-load-firmware aml-libdvr \
-- 
2.25.1

